modules/src/ics02_client/client_def.rs

```diff
@@ -106,12 +113,39 @@ impl Header for AnyHeader {
 
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub enum AnyClientState {
-    Tendermint(crate::ics07_tendermint::client_state::ClientState),
+    Tendermint(TendermintClientState),
 
     #[cfg(test)]
     Mock(MockClientState),
 }
 
+impl AnyClientState {
+    pub fn from_any(any: prost_types::Any) -> Result<Self, Error> {
+        match any.type_url.as_str() {
+            "ibc.tendermint.ClientState" => {
```

Comment:
Wondering if we should define these `type_url` strings in a separate place so they can also be used for encoding. We can do it later also.

Reply:
Agreed! See my reply to Adi’s comment above.